python/beeai_framework/backend/message.py

```diff
@@ -120,7 +130,9 @@ def __str__(self) -> str:
 
     def _verify(self, content: list[Any]) -> list[T]:
         models = self._models()
-        return [to_any_model(models, value) for value in content]
+        return [
+            self._from_string(value) if isinstance(value, str) else to_any_model(models, value) for value in content
```

Comment:
I'm a bit confused as to why this change. Afaik this is only called on line 90 above, and that line already handles `self._from_string`